code/controllers/subsystem/minimap.dm

```diff
@@ -689,6 +689,10 @@ SUBSYSTEM_DEF(minimaps)
 	if(faction == FACTION_NEUTRAL && isobserver(user))
 		faction = allowed_flags == MINIMAP_FLAG_XENO ? XENO_HIVE_NORMAL : FACTION_MARINE
 
+	if(is_xeno && xeno.hive.see_humans_on_tacmap)
+		allowed_flags |= MINIMAP_FLAG_USCM|MINIMAP_FLAG_PMC|MINIMAP_FLAG_UPP|MINIMAP_FLAG_CLF
+		targeted_ztrait = ZTRAIT_MARINE_MAIN_SHIP
```

Comment:
```suggestion
	if(is_xeno && xeno.hive.see_humans_on_tacmap && targeted_ztrait != ZTRAIT_MARINE_MAIN_SHIP)
		allowed_flags |= MINIMAP_FLAG_USCM|MINIMAP_FLAG_PMC|MINIMAP_FLAG_UPP|MINIMAP_FLAG_CLF
		targeted_ztrait = ZTRAIT_MARINE_MAIN_SHIP
		map_holder = null
```
We must also null out `map_holder` so the different z-level minimap is fetched below. The additional condition is just to make this only occur once.

But just VVing it to null I can then get it to work:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/a6915bae-80f4-40a1-9764-0b3487ac842e)